utils/spdx/src/main/resources/declared-license-mapping.yml

```diff
@@ -241,6 +241,7 @@
 "GNU General Public License, version 2": GPL-2.0-only
 "GNU General Public License, version 2, with the Classpath Exception": GPL-2.0-only WITH Classpath-exception-2.0
 "GNU LESSER GENERAL PUBLIC LICENSE V3.0": LGPL-3.0-only
+"GNU Lesser General Public License v2 (LGPLv2)": LGPL-2.1-or-later
```

Comment:
From purely looking at the mapped text (without knowing anything about your concrete package), this should map to `LGPL-2.0-or-later` IMO.